sonarts-sq-plugin/its/plugin/src/test/java/org/sonar/typescript/its/SonarLintTest.java

```diff
@@ -93,6 +93,24 @@ public void test() throws Exception {
     assertThat(logs).contains("SonarTS Server is started", "Started SonarTS Analysis");
   }
 
+  @Test
+  public void reportParsingError() throws Exception {
+    Files.write(projectDir.toPath().resolve("tsconfig.json"), "{}".getBytes(StandardCharsets.UTF_8));
+    ClientInputFile inputFile = prepareInputFile("foo.ts",
+      "function foo() {\n" +
+        "    let x = 4; \n" +
+        "    if (x = \n" +
+        "}");
+
+    final List<Issue> issues = new ArrayList<>();
+
+    StandaloneAnalysisConfiguration standaloneAnalysisConfiguration = new StandaloneAnalysisConfiguration(projectDir.toPath(), temp.newFolder().toPath(),
+      Collections.singletonList(inputFile), ImmutableMap.of());
+    AnalysisResults results = sonarlintEngine.analyze(standaloneAnalysisConfiguration, issues::add, null, null);
+    assertThat(results.failedAnalysisFiles()).containsExactly(inputFile);
```

Comment:
@saberduck can we add similar assertion to other tests? (to see that there is no failed files)